### PR TITLE
rST to markdown conversion

### PR DESCRIPTION
**Pull request by ivankocienski**
Using markdown as the document format. It's still a little sphinx dependent for doing fancy things like ToCs but that is minimal

### CODE REVIEW

File: the-team.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
-The Team
-========
+# The Team
 
 PlaceCal is currently run by Geeks for Social Change, a social
 enterprise formed by Dr Kim Foale to create community-led technical and
 social interventions.
 
-PlaceCal is developed by Geeks for Social Change and PHASE@MMU. It was
+PlaceCal is developed by Geeks for Social Change and <mailto:PHASE@MMU>. It was
```
Comment (lexiwitch) [inline the-team.md]: Not 100% sure that this shouldn't be an email link, however, it seems wrong in context haha

File: introduction/philosophy.md [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
--------------------
+## Founded on research
 
 The PlaceCal pilot emerged as part of a large scale co-research project.
 It was responding to the need for older people, community organisations
 and institutional partners to work together more effectively.
 
 This project was a collaboration between Dr Kim Foale (Geeks for Social
-Change), Prof. Stefan White (PHASE@MMU) and the Manchester Age Friendly
+Change), Prof. Stefan White (<mailto:PHASE@MMU>) and the Manchester Age Friendly
```
Comment (lexiwitch) [inline introduction/philosophy.md]: I think the same thing happened here too

**Review by lexiwitch — CHANGES_REQUESTED** [review]
looks good aside from those two mailto links tho! :+1:

**Comment by kimadactyl** [comment]
Yep, not an email link but otherwise good!

**Comment by ivankocienski** [comment]
These have been fixed now

**Comment by kimadactyl** [comment]
Superceded by #6

**Pull request closed without merging**